packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx

```diff
@@ -57,17 +57,41 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
 
     switch (item.type) {
         case ResourceViewItemType.CHART:
-            if (user.data?.ability?.cannot('manage', 'SavedChart')) {
+            if (
+                user.data?.ability?.cannot(
+                    'manage',
+                    subject('SavedChart', {
+                        organizationUuid: organizationUuid,
```

Comment:
same

Reply:
On this one the Chart and the dashboard objects dont have the orgUuid, so I'm just going to leave them out. We could add the orgUuid to those if we want to check it too. I can do that in this or the next one if you want.